enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackupCommand.java

```diff
@@ -236,6 +236,7 @@ public void execute( String[] args ) throws IncorrectUsage, CommandFailed
         {
             try
             {
+                outsideWorld.stdOutLine( "Doing consistency check..." );
```

Comment:
"Performing consistency check" seems like a better fit (I am not a native speaker though :/)